client/src/helpers/APICalls/contest.ts

```diff
@@ -1,5 +1,26 @@
+import { AuthApiData } from '../../interface/AuthApiData';
 import { FetchOptions } from '../../interface/FetchOptions';
 
+export async function createContestAPI(
+  title: string,
+  description: string,
+  prizeAmount: string,
+  deadline: Date | null,
```

Comment:
Why are we using type null here?

Reply:
remove null in API calls and component definition